jekyll/_cci2/container-runner-installation.adoc

```diff
@@ -32,7 +34,7 @@ agent:
       token: MY_TOKEN
 ```
 +
-4. Run `helm install container-agent container-agent/container-agent -n circleci`
+5. Run `helm install container-agent container-agent/container-agent -n circleci -f values.yaml`
```

Comment:
[sharing; no action needed]

I was also able to "squeeze" step 3 into step 5 via passing the `--create-namespace` flag in step 5 instead

```console
helm install container-agent container-agent/container-agent -n circleci --create-namespace -f values.yaml
```